winter/core/component.py

```diff
@@ -0,0 +1,19 @@
+import abc
+import typing
+
+from .component_method import ComponentMethod
+
+
+class Component(abc.ABC):
```

Comment:
Why ABC?

Reply:
I use register for mark classes as components